canard/blocks.css

```diff
@@ -21,7 +21,7 @@ Description: Used to style Gutenberg Blocks.
 /* Captions */
 
 [class^="wp-block-"] figcaption {
-	color: inherit;
+	color: #777;
```

Comment:
This comes from https://github.com/Automattic/themes/commit/8e8cabba8e7d96db0b9f68c5f4614d7df2dd48b8

It probably doesn't make much difference but I'd rather leave it as `inherit`a than `#777` even though I know that's what's in production.

Reply:
"Leaving it as inherit" would necessitate first pushing that change to production and cycling a build before we can complete this merge.  Is that worth it? Alternately we could flag that as an improvement to make after this process has completed, returning the value to inherit once this process is complete.